services/viva/microservice/test/lambdas/submitCompletion.test.ts

```diff
@@ -62,12 +62,32 @@ function createDependencies(
     postCompletion: () => Promise.resolve({ status: 'OK' }),
     updateCase: () => Promise.resolve(),
     getAttachments: () => Promise.resolve([]),
+    deleteAttachments: () => Promise.resolve(undefined),
     ...partialDependencies,
   };
 }
 
 it('successfully submits completions', async () => {
```

Comment:
maybe add '...and deletes attachments after success'?

Reply:
Will not change the description.